src/lib/components/Panels.js

```diff
@@ -88,13 +125,20 @@ export const PanelDisplay = (props) => {
  */
 const Panel = ({
   data,
-  graphType,
-  facetStyles={},
+  params,
+  styles=undefined,
   locations=undefined, /* optional. Defaults to all available */
 }) => {
   const {modelData, error} = data;
   const [logit, toggleLogit] = useState(false);
-  const sizes = {...useResponsiveSizing(graphType), ...facetStyles};
+  const sizes = useMemo(
+    () => ({...responsiveSizing(params), ...(styles ? styles : {})}),
+    [params, styles]
+  );
+  const canUseLogit = useMemo(
+    () => (params.canUseLogit || params.preset==="frequency"),
+    [params]
+  )
```

Comment:
Yeah, this is a really good observation. Something like `useDeepCompareMemo` is what I want to use -- I only want components to rerender when some contents (rather than the reference) have changed. This was my aim with `useMemo`, not speed per se. 

> This behavior is not visible in the example App in this repo because it doesn't have state updates that causes it to re-render.

I'll add a re-render button to the app so we can test this.

Reply:
Resolved in [3b23d2f](https://github.com/nextstrain/forecasts-viz/pull/9/commits/3b23d2f84a24760854a2b9805a49ab77657e17e7)